docs/contributing/accessibility/accessibility.mdx

```diff
@@ -0,0 +1,76 @@
+---
+sidebar_position: 5
+---
+
+# Accessibility
+
+Bitwarden strives to comply with WCAG AA requirements. The following serves as an introduction to
```

Comment:
Should we define WCAG AA as it might be an unfamiliar term to some?  Should we link to something like: https://www.w3.org/WAI/WCAG2AA-Conformance?

Reply:
Yep, I like that!